From 7d3e5e4e72d36b0886a03e4263605f2c6d1af456 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Fri, 7 Jun 2024 14:40:15 +0200
Subject: [PATCH 631/954] fix(st): update STM32MP2 part number
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The part number values aren't exact according to reference manual.
This fix updates values.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git f702ae7d631426f892fea41153828298917fb4e1 fix(st): update STM32MP2 part number]
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Change-Id: I93f5f4de51266c8c054124845d7c5e46085e4408
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/364588
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/386063
---
 plat/st/stm32mp2/stm32mp2_def.h | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/plat/st/stm32mp2/stm32mp2_def.h b/plat/st/stm32mp2/stm32mp2_def.h
index 145fd746f4..3918126db0 100644
--- a/plat/st/stm32mp2/stm32mp2_def.h
+++ b/plat/st/stm32mp2/stm32mp2_def.h
@@ -90,14 +90,14 @@
 #if STM32MP25
 #define STM32MP2_CHIP_ID	U(0x505)
 
-#define STM32MP251A_PART_NB	U(0x400B3E65)
-#define STM32MP251C_PART_NB	U(0x000B3065)
-#define STM32MP251D_PART_NB	U(0xC00B3E65)
-#define STM32MP251F_PART_NB	U(0x800B3065)
-#define STM32MP253A_PART_NB	U(0x400B2E04)
-#define STM32MP253C_PART_NB	U(0x000B2004)
-#define STM32MP253D_PART_NB	U(0xC00B2E04)
-#define STM32MP253F_PART_NB	U(0x800B2004)
+#define STM32MP251A_PART_NB	U(0x400B3E6D)
+#define STM32MP251C_PART_NB	U(0x000B306D)
+#define STM32MP251D_PART_NB	U(0xC00B3E6D)
+#define STM32MP251F_PART_NB	U(0x800B306D)
+#define STM32MP253A_PART_NB	U(0x400B3E0C)
+#define STM32MP253C_PART_NB	U(0x000B300C)
+#define STM32MP253D_PART_NB	U(0xC00B3E0C)
+#define STM32MP253F_PART_NB	U(0x800B300C)
 #define STM32MP255A_PART_NB	U(0x40082E00)
 #define STM32MP255C_PART_NB	U(0x00082000)
 #define STM32MP255D_PART_NB	U(0xC0082E00)
-- 
2.53.0

